extensions-core/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/CamelContextProcessor.java

```diff
@@ -159,6 +159,7 @@ public CamelRuntimeBuildItem runtime(
         }
 
         return new CamelRuntimeBuildItem(
-                recorder.createRuntime(beanContainer.getValue(), context.getCamelContext()));
+                recorder.createRuntime(beanContainer.getValue(), context.getCamelContext()),
+                config.bootstrap.enabled);
```

Comment:
should this option be renamed to auto-start ? 
it is perfectly fine for me to leave as it is but not sure if it is clear for evenryone

Reply:
I think we do not need to rename as long as the option docs explains the option properly, which I think it does.